src/less/components/choice-tile/index.less

```diff
@@ -1,9 +1,9 @@
-@import (reference) 'components/box/index';
```

Comment:
Can we remove this file altogether (`src/less/components/choice-tile/index.less`)?  It doesn't look like it's loaded by LESS any more.

Reply:
Changes have been done, please review.